app/upgrades.go

```diff
@@ -55,6 +56,24 @@ func (app UmeeApp) RegisterUpgradeHandlers(bool) {
 	app.registerUpgrade("v5.0", upgradeInfo, ugov.ModuleName, wasm.ModuleName)
 	app.registerUpgrade5_1(upgradeInfo)
 	app.registerUpgrade6(upgradeInfo)
+	app.registerNewTokenEmissionUpgrade(upgradeInfo)
+}
+
+// TODO: this upgrade registration is just for testing purpose, once we finalize the release for new token emission
+// then we need to change planName and storeUpgrades
+func (app *UmeeApp) registerNewTokenEmissionUpgrade(upgradeInfo upgradetypes.Plan) {
+	// TODO:finalize the name
+	planName := "token_emission"
+	app.UpgradeKeeper.SetUpgradeHandler(planName,
+		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
+		},
+	)
+
+	app.storeUpgrade(planName, upgradeInfo, storetypes.StoreUpgrades{
+		Added:   []string{metoken.ModuleName},
```

Comment:
```suggestion
		Added:   []string{/*metoken.ModuleName*/},
```
I think metoken module is still experimental flag, so if we merge this to main now, the added module can be commented out

Reply:
I will comment the this upgrade handler